index.html

```diff
@@ -122,43 +144,54 @@ <h2 class="text-primary  col-4">Prizing</h2>
         <div class="bg-light text-dark p-2">
             <br>
             <h2 class="text-center">Featured Speakers</h2>
-            <hr class="my-4 text-primary bg-primary border-5">
+            <hr>
             
-            <div class="card mb-3 border-0 mw-500">
-                <div class="d-flex g-0 flex-nowrap">
-                  <div class="card-chess" >
-                    <img class="card-speaker img-fluid rounded-start" src="https://thumbs.dreamstime.com/b/old-man-suit-talking-portrait-senior-caucasian-businessman-white-isolated-background-old-man-suit-talking-113566335.jpg" alt="Image of speaker">
-                  </div>
-                  <div >
-                    <div class="card-body">
-                      <h5 class="card-title">Card title</h5>
-                      <p class="card-text text-primary">Bekcham Professor of Entrepreneurial Legal studies ag Harvard Law School.</p>
-                      <p class="card-text">Bekcham Professor of Entrepreneurial Legal studies ag Harvard Law School.</p>
-                    </div>
-                  </div>
-                </div>
+            <!--cards container-->
+            <div class="presenters">
+                <!--Speakers loded dinamically with script.js-->
             </div>
 
-            <div class="card mb-3 border-0 mw-500">
-                <div class="d-flex g-0 flex-nowrap">
-                  <div class="card-chess" >
-                    <img class="card-speaker img-fluid rounded-start" src="https://thumbs.dreamstime.com/b/old-man-suit-talking-portrait-senior-caucasian-businessman-white-isolated-background-old-man-suit-talking-113566335.jpg" alt="Image of speaker">
-                  </div>
-                  <div >
-                    <div class="card-body">
-                      <h5 class="card-title">Card title</h5>
-                      <p class="card-text text-primary">Bekcham Professor of Entrepreneurial Legal studies ag Harvard Law School.</p>
-                      <p class="card-text">Bekcham Professor of Entrepreneurial Legal studies ag Harvard Law School.</p>
-                    </div>
-                  </div>
-                </div>
-            </div>
+
+        </div>
 
 
+        <!--div 4 dark-->
+        <div class="bg-dark text-light text-center">
+            <br>
+            <h2>Partner</h2>
+            <hr>
+            <div class="d-flex flex-wrap justify-content-center">
+                <h2 class="m-2">mozilla</h2>
+                <h2 class="m-2">Microsoft</h2>
+                <h2 class="m-2">Google</h2>
+                <h2 class="m-2">Alex Dev</h2>
+                <h2 class="m-2">Udemy</h2>
+            </div>
+            <br>
         </div>
+
+        <br>
+        <!--footer light-->
+        <footer class="bg-light text-dark">
+            <div class="footer-info">
+                <div class="me-2 footer-logo">
+                    <img class="me-1 logo" src="./assets/Logo.svg" alt="">
```

Comment:
- [ ] Please can you add an alt attribute value to this image. This will help screen readers to identify images. Please do the same for all other img elements in both the inde.html file and the about.html file too